shared/constants/chat.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import {List, Map, Record} from 'immutable'
+import {Buffer} from 'buffer'
+
+import type {ConversationID as RPCConversationID, ChatActivity, ConversationInfoLocal} from './types/flow-types-chat'
+import type {NoErrorTypedAction} from './types/flux'
+
+export type MessageType = 'Text'
+export type FollowState = 'You' | 'Following' | 'Broken' | 'NotFollowing'
+
+export type Message = {
+  type: 'Text',
```

Comment:
? this is me describing the various types.  so MessageType will actually to be a list so i think this should stay as is